<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" name="ConfigTest.TestPrefix-20150930191704" tests="3" time="0.091">
	<properties/>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
	<testcase classname="ConfigTest.TestPrefix" name="test_initial_config" time="0.023"/>
	<testcase classname="ConfigTest.TestPrefix" name="test_sensor_prefix_split" time="0.036"/>
	<testcase classname="ConfigTest.TestPrefix" name="test_move_prefix" time="0.033">
		<error message="'str' object has no attribute '_version'" type="AttributeError">
<![CDATA[Traceback (most recent call last):
  File "/home/saikappa/workspace/SciPass/python/t/ConfigTest.py", line 68, in test_move_prefix
    self.api.getBalancer("%016x" % self.datapath.id, "R&E").moveGroupPrefix("group1","group2","10.0.18.128/25")
  File "./SimpleBalancer.py", line 485, in moveGroupPrefix
    priority = self.getPrefixPriority(targetPrefix)
  File "./SimpleBalancer.py", line 140, in getPrefixPriority
    if(pfix.Contains(prefix)):
  File "/usr/lib/python2.6/site-packages/ipaddr.py", line 629, in __contains__
    if self._version != other._version:
AttributeError: 'str' object has no attribute '_version'
]]>		</error>
	</testcase>
</testsuite>
